osu.Game.Tests/Visual/Editing/TestSceneTimelineZoom.cs

```diff
@@ -13,20 +13,12 @@ public class TestSceneTimelineZoom : TimelineTestScene
         public override Drawable CreateTestComponent() => Empty();
 
         [Test]
-        [FlakyTest]
-        /*
-         * Fail rate around 0.3%
-         *
-         * TearDown : osu.Framework.Testing.Drawables.Steps.AssertButton+TracedException : range halved
-         *   --TearDown
-         *      at osu.Framework.Threading.ScheduledDelegate.RunTaskInternal()
-         *      at osu.Framework.Threading.Scheduler.Update()
-         *      at osu.Framework.Graphics.Drawable.UpdateSubTree()
-         */
         public void TestVisibleRangeUpdatesOnZoomChange()
         {
             double initialVisibleRange = 0;
 
+            AddUntilStep("wait for track load", () => Beatmap.Value.TrackLoaded);
+
```

Comment:
`Beatmap.Value.TrackLoaded` only ensures `track` is not null, but I believe that doesn't necessarily mean the track is actually loaded? `MusicController.TrackLoaded` on the other hand does check that:
```suggestion
            AddUntilStep("wait for track load", () => MusicController.TrackLoaded);
```